hub/constraints/generalities/gas.lisp

```diff
@@ -16,9 +16,7 @@
                (begin (hub-stamp-constancy GAS_EXPECTED)
```

Comment:
You may use the alias here to be completely consistent with specs. It already exists in the `lisp`, but it is not currently used:

```
(defalias
    ;;
   ...
    GAS_XPCT            GAS_EXPECTED
```

Reply:
Do you want to add an alias ?

Reply:
In the constraints.lisp file the extended version `GAS_EXPECTED` is used, even if the alias `GAS_XPCT` has been created (that is the same notation we use in the spec).

Reply:
I see

Reply:
See https://github.com/Consensys/linea-constraints/issues/330